internal/pkg/cli/deploy/workload.go

```diff
@@ -275,6 +297,9 @@ func newWorkloadDeployer(in *WorkloadDeployerInput) (*workloadDeployer, error) {
 		envSess:                  envSession,
 		store:                    store,
 		envConfig:                envConfig,
+		labeledTermPrinter: func(fw syncbuffer.FileWriter, bufs []*syncbuffer.LabeledSyncBuffer, opts ...syncbuffer.LabeledTermPrinterOption) labeledTermPrinter {
```

Comment:
This will be a`IncompatibleAssign` 🤔 . if I directly use `syncbuffer.NewLabeledTermPrinter`  because the the function signature of the `labeledTermPrinter` field in workloadDeployer struct does not match the signature of `syncbuffer.NewLabeledTermPrinter` function.